### PR TITLE
testing/polybar: new aport

### PR DESCRIPTION
**Pull request by tanertas**
https://polybar.github.io/
A fast and easy-to-use tool for creating status bars.

Full featured build including alsa, pulseaudio, network (libnl3), curl, mpd, i3wm modules.

### CODE REVIEW

**Comment by tanertas** [comment]
@Ikke  @clandmeter  @andypost Can anyone look at this?

File: testing/polybar/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+build() {
+	cd "$builddir"/build
+	cmake -DCMAKE_INSTALL_PREFIX=/usr ..
+	cmake --build .
+}
+
+check() {
+	cd "$builddir"
+	# At least we can do this without X
+	./build/bin/polybar -v > /dev/null 2>&1
```
Comment (Ikke) [inline testing/polybar/APKBUILD]: Just running `polybar -v` is not appropriate as it often gives very little coverage.

**Comment by tanertas** [comment]
@Ikke ok, remove check() completely. I thought it was better than nothing.

**Comment by Ikke** [comment]
On ppc64le:

> /home/buildozer/aports/testing/polybar/src/polybar-3.3.0/lib/i3ipcpp/src/ipc-util.cpp: In constructor 'i3ipc::buf_t::buf_t(uint32_t)':
/home/buildozer/aports/testing/polybar/src/polybar-3.3.0/lib/i3ipcpp/src/ipc-util.cpp:37:9: error: 'char* strncpy(char*, const char*, size_t)' specified bound 6 equals destination size [-Werror=stringop-truncation]
  strncpy(header->magic, g_i3_ipc_magic.c_str(), sizeof(header->magic));
  ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

**Comment by tanertas** [comment]
@jaagr we can't compile i3ipcpp on ppc64le using gcc-8.3. Have you any idea? Build log and build script is below.

https://build.alpinelinux.org/buildlogs/build-edge-ppc64le/testing/polybar/polybar-3.3.0-r0.log
https://git.alpinelinux.org/aports/tree/testing/polybar/APKBUILD

**Comment by Ikke** [comment]
Has been fixed in 52d89da4 (see included patch).

**Pull request closed without merging**